openssl/src/ssl/connector.rs

```diff
@@ -247,13 +249,16 @@ impl SslAcceptor {
     /// This corresponds to the modern configuration of version 5 of Mozilla's server side TLS recommendations.
     /// See its [documentation][docs] for more details on specifics.
     ///
-    /// Requires OpenSSL 1.1.1 or newer.
+    /// Requires OpenSSL 1.1.1 or LibreSSL 3.4.0 or newer.
     ///
     /// [docs]: https://wiki.mozilla.org/Security/Server_Side_TLS
-    #[cfg(ossl111)]
+    #[cfg(any(ossl111, libressl340))]
     pub fn mozilla_modern_v5(method: SslMethod) -> Result<SslAcceptorBuilder, ErrorStack> {
         let mut ctx = ctx(method)?;
+        #[cfg(ossl111)]
         ctx.set_options(SslOptions::NO_SSL_MASK & !SslOptions::NO_TLSV1_3);
+        #[cfg(libressl340)]
+        ctx.set_min_proto_version(Some(SslVersion::TLS1_3))?;
```

Comment:
Can't openssl use this as well?

Reply:
Why can't LibreSSL use the set_options approach?

Reply:
`NO_SSL_MASK` doesn't exist in LibreSSL.

Reply:
Is LibreSSL no longer just trying to be an "improved" implementation of the OpenSSL API? If it is going to go off and do its own thing, it is not clear to me that this crate should continue supporting it.

Reply:
According to the front page, the goals have not changed.

Reply:
@sfackler I would hope supporting it so that OpenBSD and other libressl users don't have unnecessary blockers for using rust would be reason enough.